src/LBFactory.sol

```diff
@@ -252,100 +280,100 @@ contract LBFactory is PendingOwnable, ILBFactory {
 
     /// @notice Set the LBPair implementation address
     /// @dev Needs to be called by the owner
-    /// @param _LBPairImplementation The address of the implementation
-    function setLBPairImplementation(address _LBPairImplementation) external override onlyOwner {
-        if (ILBPair(_LBPairImplementation).getFactory() != this) {
-            revert LBFactory__LBPairSafetyCheckFailed(_LBPairImplementation);
+    /// @param newLBPairImplementation The address of the implementation
+    function setLBPairImplementation(address newLBPairImplementation) external override onlyOwner {
+        if (ILBPair(newLBPairImplementation).getFactory() != this) {
+            revert LBFactory__LBPairSafetyCheckFailed(newLBPairImplementation);
         }
 
-        address _oldLBPairImplementation = LBPairImplementation;
-        if (_oldLBPairImplementation == _LBPairImplementation) {
-            revert LBFactory__SameImplementation(_LBPairImplementation);
+        address _oldLBPairImplementation = _LBPairImplementation;
+        if (_oldLBPairImplementation == newLBPairImplementation) {
+            revert LBFactory__SameImplementation(newLBPairImplementation);
         }
 
-        LBPairImplementation = _LBPairImplementation;
+        _LBPairImplementation = newLBPairImplementation;
 
-        emit LBPairImplementationSet(_oldLBPairImplementation, _LBPairImplementation);
+        emit LBPairImplementationSet(_oldLBPairImplementation, newLBPairImplementation);
     }
 
-    /// @notice Create a liquidity bin LBPair for _tokenX and _tokenY
-    /// @param _tokenX The address of the first token
-    /// @param _tokenY The address of the second token
-    /// @param _activeId The active id of the pair
-    /// @param _binStep The bin step in basis point, used to calculate log(1 + binStep)
-    /// @return _LBPair The address of the newly created LBPair
-    function createLBPair(IERC20 _tokenX, IERC20 _tokenY, uint24 _activeId, uint16 _binStep)
+    /// @notice Create a liquidity bin LBPair for tokenX and tokenY
+    /// @param tokenX The address of the first token
+    /// @param tokenY The address of the second token
+    /// @param activeId The active id of the pair
+    /// @param binStep The bin step in basis point, used to calculate log(1 + binStep)
+    /// @return pair The address of the newly created LBPair
+    function createLBPair(IERC20 tokenX, IERC20 tokenY, uint24 activeId, uint8 binStep)
         external
         override
-        returns (ILBPair _LBPair)
+        returns (ILBPair pair)
     {
+        // TODO: fix stack too deep to cache owner
         // address _owner = owner();
-        if (!creationUnlocked && msg.sender != owner()) revert LBFactory__FunctionIsLockedForUsers(msg.sender);
+        if (!_creationUnlocked && msg.sender != owner()) revert LBFactory__FunctionIsLockedForUsers(msg.sender);
 
-        address _LBPairImplementation = LBPairImplementation;
+        address LBPairImplementation_ = _LBPairImplementation;
```

Comment:
I think we could move to `lbPair` for variable names. I don't think `X_` is necessary here

Reply:
Definitely